migrations/versions/0226_new_letter_rates.py

```diff
@@ -0,0 +1,54 @@
+"""empty message
+
+Revision ID: 0226_new_letter_rates
+Revises: 0225_another_letter_org
+
+"""
+
+revision = '0226_new_letter_rates'
+down_revision = '0225_another_letter_org'
+
+import uuid
+import pytz
+from datetime import datetime
+from alembic import op
+
+
+start = datetime(2018, 10, 1, 0, 0, tzinfo=pytz.utc)
+
+NEW_RATES = [
+    (uuid.uuid4(), start, 1, 0.30, True, 'second'),
```

Comment:
We only need to add in the rows for 1st class letters, and the rows for 2nd class letters where the rate is going to change. For example, the rate for 1 page letters that are crown and 2nd class will not change, so we don't need to add a new row.

Reply:
Ok, there was one rate that did not change, I deleted it from the new migration so it will still be covered by the rate set earlier ^_^